### ISSUE TITLE
[Efficiency] User encounters unnecessary repetition or redundancy.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#75985]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** govcio-vfep
**Product:** GI Bill Comparison Tool
**Feature:** Specialized Mission Filter

## Findings details
**VA.gov Experience Standard - issue:** User encounters unnecessary repetition or redundancy.
**VA.gov Experience Standard - category:** Efficiency
**Launch-blocking:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When searching by location, a large number of results can lead to an impractical number of tab stops for keyboard users attempting to compare schools.

Keyboard-only users primarily navigate through a page by pressing [tab] and [shift] + [tab] to move forward and backward through the interactive elements of the page (links, buttons, form inputs). It's important for interfaces to be designed with an awareness of what's physically reasonable to ask of a user in order to complete a task.

<details><summary>Link, screenshot or steps to recreate</summary>

Steps to reproduce:

1. [Navigate to the GI Bill Comparison Tool on staging](https://staging.va.gov/education/gi-bill-comparison-tool/).
2. Switch to the "Search by location" tab.
3. Enter ZIP code 60625 (Chicago) and search.
4. Using keyboard only (pressing [tab] to move between elements), select the first three results to compare.
5. Press [tab] until focus moves to the Compare button to compare these three schools.

Note that you'll need to press [tab] 1,520 times to complete step 5.

</details>

## Recommended action
This is an issue I've flagged in previous reviews of the product, and will continue to flag. This is not a reasonable user experience for keyboard-only users. Fixing this was likely out of scope for this iteration of work on the product, but it really should be prioritized moving forward. Some possibilities include introducing some kind of pagination, or adding some kind of skip link to skip to the tray at the bottom of the page, or a more comprehensive redesign.

## References

- **WCAG Success Criteria:** N/A
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** Platform OCTO leadership asks that we document all issues we identify. If you believe that this issue is out of scope or not your team's responsibility, it is still your responsibility to resolve the issue. Please comment on Slack in your team channel tagging `@platform-governance-team-members`, providing an explanation and who you believe is responsible. Governance team will research and follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
